src/node/clean-up-temp-files.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+'use strict';
+
+const path = require('path');
+const del = require('del');
+const pTry = require('p-try');
+
+const cleanUpTempFiles = (
+  batfishConfig: BatfishConfiguration
+): Promise<void> => {
+  const assetsDirectory = path.join(
+    batfishConfig.outputDirectory,
+    batfishConfig.publicAssetsPath
+  );
+
+  const assetsJsonPath = path.join(assetsDirectory, 'assets.json');
+  const renderStaticPagesPath = path.join(
+    assetsDirectory,
+    'static-render-pages.js'
+  );
+  const renderStaticPagesMapPath = path.join(
+    assetsDirectory,
+    'static-render-pages.js.map'
+  );
+
+  return pTry(() => {
+    return del(
+      [assetsJsonPath, renderStaticPagesPath, renderStaticPagesMapPath],
+      { force: true }
+    );
```

Comment:
You could give this `del` call a `cwd` argument and then use relative paths. That way you won't have to construct absolute paths and can simplify the code here.

Reply:
woohoo! forgot about the `cwd` argument. that makes things much more serene.